llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -2422,10 +2531,64 @@ Expected<uint64_t> MCCASReader::materializeSection(cas::ObjectRef ID,
     Stream->write_zeros(1);
     return *Size + 1;
   }
+  if (auto F = AddendsRef::Cast(*Node))
+    // AddendsRef is already handled when materializing Atoms, skip.
+    return 0;
   return createStringError(inconvertibleErrorCode(),
                            "unsupported CAS node for atom");
 }
 
+/// This helper function reconstructs the current Section as it was before the
+/// CAS was created. The CAS split the section into multiple *FragmentRef or
+/// MergedFragmentRef blocks, and an AddendsRef block that contains the
+/// relocation addends for that section. The \p SectionBuffer will contain the
```

Comment:
will contain -> contains

or are these OUT arguments?

Reply:
The SectionBuffer is an out argument yes